docs/plugins/drupal-plugin.md

```diff
@@ -5,8 +5,9 @@ class: documentation
 permalink: /plugins/drupal-plugin/
 published: true
 ---
-The Drupal plugin provides an easy way for you to configure your Drupal build and to save you some shell scripting. To use the Drupal plugin you must declare `plugin: Drupal` in your `.probo.yaml` file. The plugin's parameters can automate your Drupal build by reverting features, running database updates, clearing caches, or performing other build configuration steps.
+The Probo Drupal plugin provides an easy way to set Drupal site configuration options in a Probo Build to quickly integrate a Drupal website's repository. To use the Probo Drupal plugin you must declare `plugin: Drupal` in your `.probo.yaml` file. The Drupal plugin's parameters can automate some steps specific to Drupal such as reverting features, running database updates, clearing caches, or performing other build configuration steps. 
 
+The Probo Drupal plugin inherits all [Probo LAMP plugin](/plugins/lamp-plugin/) configuration options. This allows additional Probo Build steps in your `.probo.yaml` file to layer additional LAMP configuration options and commands on top of the Drupal site specific configuration.
```

Comment:
You use "additional" twice in the same sentence, which sounds a little funny.

"This allows ~additional~ **other** Probo Build steps in your `.probo.yaml` file to layer additional LAMP configuration options and commands on top of the Drupal site specific configuration."